buildkitd/dockerd-wrapper.sh

```diff
@@ -48,6 +48,29 @@ execute() {
     fi
     mkdir -p "$EARTHLY_DOCKERD_DATA_ROOT"
 
+    if [ -f "/sys/fs/cgroup/cgroup.controllers" ]; then
+        if [ "$EARTHLY_DOCKER_WRAPPER_DEBUG" = "1" ]; then
+            echo >&2 "detected cgroups v2"
+        fi
+
+        # move script to separate cgroup, to prevent the root cgroup from becoming threaded (which will prevent systemd images (e.g. kind) from running)
+        mkdir /sys/fs/cgroup/dockerd-wrapper
```

Comment:
will this be helpful or unexpected?
```suggestion
        mkdir -p /sys/fs/cgroup/dockerd-wrapper
```

Reply:
`/sys/fs/cgroup` is created by the OS, it's not a regular directory but instead an API into the kernel, if this directory doesn't exist, then the API (i.e. cgroups) is not available, and creating a regular directory in it's place would have no effect.